packages/glibc.rb

```diff
@@ -263,9 +283,35 @@ def self.install
     FileUtils.rm Dir.glob("#{CREW_DEST_LIB_PREFIX}/libmount.so.*")
   end
 
- #  def self.check
- #   Dir.chdir 'glibc_build' do
- #     system "make -k -j#{CREW_NPROC} check"
- #   end
- # end
+  # def self.check
+  # Dir.chdir 'glibc_build' do
+  # system "make -k -j#{CREW_NPROC} check"
+  # end
+  # end
+  def self.postinstall
+    @libraries = %w[ld libBrokenLocale libSegFault libanl libc libcrypt
+                    libdl libm libmemusage libmvec libnsl libnss_compat libnss_db
+                    libnss_dns libnss_files libnss_hesiod libpcprofile libpthread
+                    libresolv librlv librt libthread_db-1.0 libutil]
+    Dir.chdir CREW_LIB_PREFIX do
+      @libraries.each do |lib|
+        Dir.glob("/#{ARCH_LIB}/#{lib}*").each do |f|
+          if `file #{f} | grep "shared object"`
+            g = File.basename(f)
+            FileUtils.ln_sf f.to_s, g.to_s
+          end
+        end
+      end
+      if LIBC_VERSION == '2.32'
+        FileUtils.ln_sf "/#{ARCH_LIB}/libresolv-#{LIBC_VERSION}.so", 'libresolv.so.2'
+        FileUtils.ln_sf "/#{ARCH_LIB}/libc.so.6", '.'
+        case ARCH
+        when 'armv7l', 'aarch64'
+          FileUtils.ln_sf "/#{ARCH_LIB}/ld-linux-armhf.so.3", 'ld-linux-armhf.so.3'
+        when 'x86_64'
+          FileUtils.ln_sf "/#{ARCH_LIB}/ld-linux-x86-64.so.2", 'ld-linux-x86-64.so.2'
+        end
+      end
```

Comment:
Could the symlinks fix issues with the 2.32 binaries?

Reply:
They do not. I think the issue is with the crt binaries bring generated.

Reply:
They do not. I think the issue is with the crt binaries bring generated.